plugins/sources/frontend/public/stylesheets/sources.scss

```diff
@@ -0,0 +1,87 @@
+.sources-tab {
```

Comment:
We might use a more specific namespace. I mean it feels like "sources-tab" can be coincidentally used somewhere else to stylize some other (and completely irrelevant) "sources-tab".